parser.py

```diff
@@ -0,0 +1,45 @@
+# Author: Pankil Shah
+# Created on: 03/13/2021
+# Description: Parse a corpus and dump the sentences into the database 
+
+from nltk.corpus import brown
```

Comment:
Can we add comments these, so developers who don't know what nltk is can know what it is?

Reply:
This one is an unused import which I will remove and add description for nltk.data which is used